source/class/qx/tool/cli/commands/Test.js

```diff
@@ -47,36 +45,10 @@ qx.Class.define("qx.tool.cli.commands.Test", {
     CONFIG_FILENAME : "compile-test.json",
 
     YARGS_BUILDER: {
-      verbose: {
```

Comment:
Why did you remove all these parameters? Are they duplicates?

Reply:
Because they are special to the testtapper package. So testapper should add them. With the new api this should be possible as john said